.github/workflows/test.yaml

```
@@ -0,0 +1,22 @@
+name: test
+on: 
+  pull_request:
+    branches:
+      - v3
```

What about enabling actions for any PRs and the default branch: `v3` ?

```suggestion
on:
  push:
    branches:
      - v3
  pull_request:
```

---

sgtm, added